src/registry/registry.json

```diff
@@ -1679,18 +1679,27 @@
       "suffix": "bot",
       "directoryName": "bots",
       "inFolder": false,
+      "strictDirectoryName": true,
       "children": {
         "types": {
           "botversion": {
             "id": "botversion",
             "name": "BotVersion",
-            "directoryName": "botVersions",
+            "directoryName": "",
```

Comment:
This is so that it places `Bot` and `BotVersions` in the same directory, `force-app/main/default/bots/<bot-name>/` with a value here it would place `BotVersions` in `force-app/main/default/bots/<bot-name>/botVersions/`